api/source/service/STIGService.js

```diff
@@ -232,7 +232,7 @@ exports.queryBenchmarkRules = async function ( benchmarkId, revisionStr, inProje
   
   if (revisionStr != 'latest') {
     joins = ['revision rev']
-    let [input, version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
+    let [version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
```

Comment:
regex returns what it captures positionally, so we need "input" here, even though it is not used.